roles/nginx/tasks/install.yml

```diff
@@ -24,9 +24,9 @@
     src: "{{ item.src }}"
     dest: "{{ item.dest }}"
   with_items:
-    - { src: 'roles/nginx/templates/server.conf', dest: '/etc/nginx/' }
-    - { src: 'roles/nginx/templates/nginx.conf', dest: '/etc/nginx/' }
-    - { src: 'roles/nginx/templates/ports.conf', dest: '/etc/{{ apache_service }}/' }
+    - { src: 'roles/nginx/templates/server.conf.j2', dest: '/etc/nginx/' }
+    - { src: 'roles/nginx/templates/nginx.conf.j2', dest: '/etc/nginx/' }
+    - { src: 'roles/nginx/templates/ports.conf.j2', dest: '/etc/{{ apache_service }}/' }
```

Comment:
need to use the file name when writing the file name.
dest ... /etc/nginx/server.conf
dest ... /etc/nginx/nginx.conf
dest ... /etc/{{ apache_service }}/ports.conf